core-java-modules/pom.xml

```diff
@@ -177,6 +177,7 @@
         <module>core-java-collections-maps-6</module>
         <module>core-java-records</module>
         <module>core-java-9-jigsaw</module>
+        <module>core-java-collections-array-list-2</module>
```

Comment:
Move this line up within the modules section so that it appears in alphabetical order, right after core-java-collections-array-list.